dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/optimizers/Optimizers.scala

```diff
@@ -0,0 +1,15 @@
+package me.shadaj.scalapy.tensorflow.api.keras.optimizers
+
+import me.shadaj.scalapy.tensorflow.api.Tensor
+import me.shadaj.scalapy.tensorflow.keras.optimizers.{Optimizers => PyOptimizers}
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.{PythonModule, PythonType}
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonOption
+import scala.language.implicitConversions
+
+class Optimizers private[api] (val underlying: PyOptimizers) extends PythonType[PyOptimizers] with PythonModule {
+  // def Adadelta(): Adadelta = py.native
+
+  // def Adam(learning_rate: Double): Adam = py.native
+
+  def SGD(learningRate: Double, momentum: Option[Double]): SGD = new SGD(underlying.SGD(learningRate, new PythonOption[Double].option2PyOption(momentum)))
```

Comment:
See the comment to `PythonOption`, I think we can get rid of the explicit `new PythonOption` (which is kinda against the idea of implicit conversion) when `PythonOption` is made an object... unless Scala 3 changed sth dramatically in this respect

Reply:
ehh the problem is the `[Double]` parameter, If I can't figure it out then it won;t be an implicit conversion.

Reply:
Problem solved. As in `PythonOption`